LayoutTests/http/tests/cache/cache-match-invalid-url.html

```diff
@@ -0,0 +1,19 @@
+<script>
```

Comment:
This test should probably go into LayoutTests/http/tests/cache-storage, not cache/